graylog2-web-interface/src/components/widgets/Widget.jsx

```diff
@@ -216,10 +216,10 @@ const Widget = React.createClass({
                              onUpdate={this.updateWidget} />
     );
 
-    let disabledTooltip = null;
+    let disabledReplay = false;
     if (this.props.streamIds != null && this.props.widget.config.stream_id &&
         !this.props.streamIds[this.props.widget.config.stream_id]) {
-      disabledTooltip = 'The stream is not available, cannot replay search.';
+      disabledReplay = true;
```

Comment:
FWIW, I would change this to 
`const disabledReplay = this.props.streamIds != null && this.props.widget.config.stream_id && !this.props.streamIds[this.props.widget.config.stream_id];`

to get rid of the `let`/`if` and make it a bit more concise.